core/settings.py

```diff
@@ -69,6 +72,12 @@
     },
 }
 
+ELASTICSEARCH_DSL = {
+    'default': {
+        'hosts': 'http://localhost:9200'
+    },
+}
```

Comment:
Consider referencing the host from an environment variable for flexibility between dev/staging/prod. For example:
```
'hosts': os.environ.get('ELASTICSEARCH_DSL_HOST', 'http://localhost:9200')
```